package-lock.json

```diff
@@ -0,0 +1,12555 @@
+{
```

Comment:
Please remove this file and add it to `.gitignore`.

Reply:
Done!